src/galasm.c

```diff
@@ -575,18 +575,6 @@ int AssemblePldFile(char *file, struct Config *cfg)
                                 }
                             }
                         }
-
```

Comment:
Deletion of this block of code looks like it might have unintended consequences for gal_type other than GAL16V8 or GAL20V8, i.e. GAL20RA10 or GAL22V10 - Jedec.GALSYN and Jedec.GALAC0 could be left uninitialized?